Lib/traceback.py

```diff
@@ -133,35 +132,12 @@ def print_exception(exc, /, value=_sentinel, tb=_sentinel, limit=None, \
 
 BUILTIN_EXCEPTION_LIMIT = object()
 
-def _can_colorize():
-    if sys.platform == "win32":
-        try:
-            import nt
-            if not nt._supports_virtual_terminal():
-                return False
-        except (ImportError, AttributeError):
-            return False
-
-    if os.environ.get("PYTHON_COLORS") == "0":
-        return False
-    if os.environ.get("PYTHON_COLORS") == "1":
-        return True
-    if "NO_COLOR" in os.environ:
-        return False
-    if not _COLORIZE:
-        return False
-    if "FORCE_COLOR" in os.environ:
-        return True
-    if os.environ.get("TERM") == "dumb":
-        return False
-    try:
-        return os.isatty(sys.stderr.fileno())
-    except io.UnsupportedOperation:
-        return sys.stderr.isatty()
 
 def _print_exception_bltin(exc, /):
+    from _colorize import can_colorize
```

Comment:
Why not import this at the module level along with `ANSIColors`?

Reply:
Will move it.

Was originally thinking it's only used in this one place, so don't need to import most of the time. But then couldn't do the same with `ANSIColors`, so they might as well both be together at the top.